.gitlab-ci.yml

```diff
@@ -14,12 +14,13 @@ before_script:
   stage: build
   tags:
     - deploy_test
+  interruptible: true
```

Comment:
It looks unrelated here.

Reply:
This is the hidden job used as template for all jobs. The main idea is to skip building jobs in queue if the newer same jobs exists.

Reply:
I understand, but it does not look related here anyway. This PR is about apline version bump, right?